frontend/src/modules/schemas/editor/components/content/base/GroupRenderer.ts

```
@@ -0,0 +1,209 @@
+          return getContentNodes([child.component.subTree]);
+        }
+        return child.children?.length ? getContentNodes(child.children as VNode[]) : [];
+      });
+    }
+
+    function getChildrenClientRect(): NodeRect {
+      if (!nodes.length) {
+        return NodeRect.BLANK;
+      }
+
+      let minX = Infinity;
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)